redis/commands/json/commands.py

```diff
@@ -121,7 +121,7 @@ def objkeys(
         """  # noqa
         return self.execute_command("JSON.OBJKEYS", name, str(path))
 
-    def objlen(self, name: str, path: Optional[str] = Path.root_path()) -> int:
+    def objlen(self, name: str, path: Optional[str] = Path.root_path()) -> List[Union[int, None]]:
```

Comment:
```suggestion
    def objlen(self, name: str, path: Optional[str] = Path.root_path()) -> List[Optional[int]]:
```

Reply:
Hello I was just copying the type hint of the return value of `arrlen` and it seems several other methods in https://github.com/redis/redis-py/blob/5391c5fc39a2c9aaa1b2e5acc799b93444270b88/redis/commands/json/commands.py use `List[Union[int, None]]` so is it better to keep it in line? Well of course you have the final say. If you insist `List[Optional[int]]` I'll commit.